R/module_teal_data.R

```diff
@@ -116,9 +115,10 @@ srv_validate_reactive_teal_data <- function(id, # nolint: object_length
             need(
               FALSE,
               paste(
-                strip_style(data_out$message),
+                "Shiny error when executing the `data` module:",
+                strip_style(paste(data_out$message, collapse = "\n")),
```

Comment:
Not needed as `data_out$message == ""`  always - because of the condition: `if (inherits(data_out, "shiny.silent.error") && identical(data_out$message, ""))`

```suggestion
                "Shiny error when executing the `data` module",
```